appveyor.yml

```diff
@@ -0,0 +1,57 @@
+environment:
+
+  global:
+     SDL_VIDEODRIVER: "dummy"
+     SDL_AUDIODRIVER: "disk"
+
+
+  matrix:
+    - PYTHON: "C:\\Python27"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python34"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python35"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python36"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python27-x64"
+      PYTHON_ARCH: "64"
+
+    - PYTHON: "C:\\Python34-x64"
+      PYTHON_ARCH: "64"
+
+    - PYTHON: "C:\\Python35-x64"
+      PYTHON_ARCH: "64"
+
+    - PYTHON: "C:\\Python36-x64"
+      PYTHON_ARCH: "64"
+
+init:
+  - "ECHO %PYTHON% %PYTHON_ARCH%"
+
+install:
+  # Ensure we use the right python version
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "python --version"
+  - "powershell appveyor\\install.ps1"
+  - "pip install -e ."
+  - ps: "ls"
+  - "powershell appveyor\\copydlls.ps1"
+  - ps: "ls pygame"
+  - "python setup.py bdist_wheel"
+  - ps: "ls dist"
+
+
+# Appveyor's build step is specific to .NET projects, so we build in the
+# install step instead.
+build: false
+
+test_script:
+  - "python -m test"
+
+#artifacts:
+#  - path: dist\*

```

Comment:
My plan is to have appveyor build wheels for us. It already runs bdist_wheel, but I still need to work out how to include the required dlls. Once that's in place, I'll uncomment this so we can grab the produced wheels.